src/VisualLinter/Linting/LinterMessage.cs

```diff
@@ -0,0 +1,68 @@
+﻿using jwldnr.VisualLinter.Enums;
+
+namespace jwldnr.VisualLinter.Linting
+{
+    public class LinterMessage
```

Comment:
While it gets the job done-- I don't like how the `LinterMessage.cs` class is shared by both eslint and stylelint because the properties aren't the same.

For example constructor for `EslintMessage` has the following:
`if (severity == 1) this.Severity = RuleSeverity.Warn;`

while constructor for `StylelintMessage` has the following:
`if (severity == "warning") this.Severity = RuleSeverity.Warn;`

They should be separate classes with different `LinterType`'s. 😄